test/assets/serviceworkers/fetch/sw.html

```diff
@@ -1,4 +1,5 @@
 <link rel="stylesheet" href="./style.css">
 <script>
   window.registrationPromise = navigator.serviceWorker.register('sw.js');
```

Comment:
There is a second test that uses this pattern, and we dont need window.registrationPromise anymore.

Reply:
It's needed to unregister!